lms/djangoapps/courseware/views/views.py

```diff
@@ -576,7 +574,7 @@ def handle_exceptions(request, course, exception):
             request.path,
             getattr(user, 'real_user', user),
             user,
-            None if course is None else text_type(course.id),
+            text_type(course_key),
```

Comment:
could you clarify this change?

Reply:
This was handling the case where this function was called w/o a course object. This PR changes the function to take a `course_key`, which is never `None`.